curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md

```diff
@@ -35,9 +35,9 @@ tests:
     testString: assert.deepEqual(translatePigLatin("algorithm"), "algorithmway", '<code>translatePigLatin("algorithm")</code> should return "algorithmway".');
   - text: <code>translatePigLatin("eight")</code> should return "eightway".
     testString: assert.deepEqual(translatePigLatin("eight"), "eightway", '<code>translatePigLatin("eight")</code> should return "eightway".');
-  - text: Should handle words where the first vowel comes in the end of the word.
+  - text: Should handle words where the first vowel comes in the middle of the word.  <code>translatePigLatin("schwartz")</code> should return "artzschway". 
     testString: assert.deepEqual(translatePigLatin("schwartz"), "artzschway", 'Should handle words where the first vowel comes in the end of the word.');
```

Comment:
```suggestion
    testString: assert.deepEqual(translatePigLatin("schwartz"), "artzschway");
```

Reply:
@beansprout Could you please expedite @Manish-Giri 's suggestion so we can get this merged